src/app/(auth)/signup/_funnel/ProfileInfo.tsx

```diff
@@ -0,0 +1,133 @@
+"use client";
+
+import SignupHeader from "@/src/components/signup/SignupHeader";
+import SignupTitle from "@/src/components/signup/SignupTitle";
+import FixedBottomButton from "@/src/components/signup/FixedBottomButton";
+import CustomDropdown from "@/src/components/signup/CustomDropdown";
+import TextInput from "@/src/components/signup/TextInput";
+import GenderSelect from "@/src/components/signup/GenderSelect";
+import AddressInput from "@/src/components/signup/AddressInput";
+import ProfileImageUpload from "@/src/components/signup/ProfileImageUpload";
+import { useSignupStore } from "@/src/stores/useSignupStore";
+import { formatBirthDate, convertImageToBase64 } from "@/src/utils/validation";
+import { SIGNUP_STEPS, SIGNUP_MESSAGES } from "@/src/constants/signup";
+import type { SignupStepProps } from "@/src/types/signup";
+
+interface StepProfileInfoProps extends SignupStepProps {
+  goPrev: () => void;
+}
+
+const StepProfileInfo: React.FC<StepProfileInfoProps> = ({ goPrev, goNext, isSocial = false }) => {
+  const {
+    role,
+    nickname,
+    gender,
+    birthDate,
+    storeName,
+    address,
+    detailAddress,
+    category,
+    profileImage,
+    setField,
+    setProfileImage,
+  } = useSignupStore();
+
+  // 프로필 이미지 업로드
+  const handleImageUpload = async (file: File) => {
+    const base64 = await convertImageToBase64(file);
+    setProfileImage(base64);
+  };
+
+  // 주소 검색 결과 처리
+  const handleAddressSearch = (fullAddress: string) => {
+    setField("address", fullAddress);
+  };
+
+  // 생년월일 입력 처리
+  const handleBirthDateChange = (value: string) => {
+    setField("birthDate", formatBirthDate(value));
+  };
+
+  const isDesigner = role === "designer";
+
+  // 폼 유효성 검사
+  const isFormValid = isDesigner
+    ? profileImage && nickname && gender && birthDate && storeName && address && category
+    : profileImage && nickname && gender && birthDate;
```

Comment:
프로필 이미지는 필수 아닌걸로 알고 있습니다! 디코에 물어본거 대답오면 확정 지읍시당
```suggestion
  // 폼 유효성 검사
  const isFormValid = isDesigner
    ? nickname && gender && birthDate && storeName && address && category
    : nickname && gender && birthDate;
```